src/lumigo_tracer/event/event_trigger.py

```diff
@@ -256,6 +258,38 @@ def _parse_dynamomdb_event(event) -> Dict[str, Union[int, List[str]]]:
     }
 
 
-def _parse_sqs_event(event) -> Dict[str, Union[int, List[str]]]:
-    mids = [record["messageId"] for record in event["Records"] if record.get("messageId")]
-    return {MESSAGE_IDS_KEY: mids} if len(mids) > 1 else {MESSAGE_ID_KEY: mids[0]}
+def _parse_sqs_event(event) -> Dict[str, Union[int, str, List[str], List[Dict[str, str]]]]:
+    message_ids = []
+    chained_resources: List[Dict[str, str]] = []
+    for record in event.get("Records", []):
+        record_message_id = None
+        if record.get("messageId"):
```

Comment:
IMO it will be nicer to do:
```
if record.get("messageId"):
    continue
```
and then you won't have to do `if record_message_id` later.